src/utils.js

```diff
@@ -543,6 +543,16 @@ const getDatasetName = (components, operators) => {
   return datasetName;
 };
 
+/**
+ * Sleep method
+ *
+ * @param {int} milliseconds
+ * @returns {Promise}
+ */
+const sleep = (milliseconds) => {
+  return new Promise((resolve) => setTimeout(resolve, milliseconds));
+};
+
```

Comment:
@shiguemi32 você usou este método? não achei chamada pra ele.

Reply:
Usei pra testa e ja dexei ai se for precisa usa mais pra frente